src/DI/SentryExtension.php

```diff
@@ -40,12 +40,16 @@ public function loadConfiguration()
         $this->getContainerBuilder()
             ->addDefinition($this->prefix('logger'))
             ->setFactory(SentryLogger::class, [Debugger::$logDirectory])
+            ->addSetup('setTracesSampleRate',
```

Comment:
Please move this setup after `register` call. Logically, the `register` call should happen first and optional setters should happen afterwards. Let's keep it that way here too.

Reply:
I am not sure if understand - sentry `init` function is initialised in `register` method - so I have to defined `traces_sample_rate` param before this method. 
I think in your case traces_sample_rate param will be always default variable